.github/workflows/ci.yaml

```diff
@@ -14,13 +14,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
       - name: Use Node.js v18
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
           node-version: 18
       - name: Install Packages
-        run: npm install
+        run: npm i
```

Comment:
This seems to work solves the `Module '"@prisma/client"' has no exported member 'PrismaClient'` error.

It looks like this is probably "fixing" it => https://github.com/prisma/prisma/blob/f718439571a60c91d3f5440226fe71031c84fa58/packages/client/package.json#L48

Maybe we `npm i @prisma/client & npx prisma generate`  and `npm i --ignore-scripts`?

Its only getting hit on the typecheck...